ext/drx/drx.c

```diff
@@ -220,12 +243,12 @@ drx_init(void)
     if (!drmgr_register_restore_state_ex_event_ex(drx_event_restore_state,
                                                   &fault_priority))
         return false;
-    if (!drmgr_register_thread_init_event(drx_thread_init) ||
-        !drmgr_register_thread_exit_event(drx_thread_exit))
-        return false;
     tls_idx = drmgr_register_tls_field();
     if (tls_idx == -1)
         return false;
+    if (!drmgr_register_thread_init_event(drx_thread_init) ||
```

Comment:
I agree this is a nop because the callbacks won't run until the init code is done. But it seems more readable to set `tls_idx` first and then register the callbacks that use it.

Reply:
> But it seems more readable to set tls_idx first and then register the callbacks that use it.

I mean, a new contributor might wonder if the callbacks may access `tlx_idx` uninitialised. Just seemed better to reorder it.